va/validation-authority.go

```diff
@@ -208,6 +237,15 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	z := sha256.Sum256(RS)
 	zName := fmt.Sprintf("%064x.acme.invalid", z)
 
+	// Check for resolver SERVFAIL for A/AAAA records
```

Comment:
This is also needed for #470, but I wouldn't be *entirely* opposed to removing it from this PR and opening a new one specifically for #470, #488, #440, and this.